cobalt/site/docs/development/setup-android.md

```diff
@@ -33,6 +33,11 @@ return and complete the following steps.
 
     If prompted, read and accept the license agreement.
 
+1.  Install JDK 17. google it for how to. Verify java version is 17 and above.
```

Comment:
I tried following the directions here but it was not obvious what instructions I should follow in my Google search. We should either provide detailed step-by-step instructions or general directions. Something like:

`Install JDK 17 and make sure it's before any other JDK on your PATH.`

I don't think we need to provide more details than this. If anything add a link to the Oracle download page as we do above for Android Studio. Either https://www.oracle.com/java/technologies/downloads/archive/ or https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html.

Reply:
done